packages/api-browser/window.js

```diff
@@ -23,4 +33,14 @@ const objectToFeaturesString = (features) => {
   }).join(',');
 };
 
+// if we have an opener, we are not the parent so we need to add it as a window
+if (window.opener) {
```

Comment:
I'm not sure about this living here, feels like it's part to the initialisation/bootstrapping process; probably fine for now, but I think we should consider moving this in future.